### ISSUE TITLE
Should trigger / pointing information be stored into in dl0?

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/ctapipe#2374]
At the moment we write the trigger information into the `/dl1/event/{subarray,telescope}` groups.

However, this is really dl0 information and we do not apply any correction which would justify making it dl1.

I guess this comes from "historic" times, where we wrote "pure" dl1 files and the data model document was not yet written.

I would propose that when we solve https://github.com/cta-observatory/ctapipe/issues/2139 we also change the data writer to write this into `/dl0/` instead.

**Comment by kosack** [comment]
Agreed. It makes more sense in DL0, and there i always a chance in the future we would want to apply some corrections, in which case there would also be a DL1 version. So it's good to separate these. 

Also now that it's clear that data products can contain data from multiple data levels, it's fine to have DL0 in a file along with DL1

**Comment by maxnoe** [comment]
Same for pointing
